ops/rak_button_firmware.ahk

```diff
@@ -0,0 +1,184 @@
+; Created 2022-07
+; by Peter Scholtens and Kerry Wang
+; Instructions can be found here: https://app.clickup.com/2434616/v/dc/2a9hr-2261/2a9hr-6207
```

Comment:
I also made a couple of comments on https://app.clickup.com/2434616/v/dc/2a9hr-2261/2a9hr-6207